_docs_explore/61-runtime.md

```diff
@@ -94,7 +94,7 @@ The Module is responsible for:
 - taking action when the User decides (accepts or rejects) a Request
   - when the Request came from a RelationshipTemplate the Module creates a Relationship with the contents of the User's Response if the User accepted the Request (rejection is currently not handled)
   - when the Request came from a Message the Module sends back a Message containing the User's Response (accept and reject)
-- listen for an incoming Relationship to create a Request out of the RelationshipTemplate that was used to create the Relationship and to directly complete the Request using the Response sent with the RelationshipCreationChange
+- listen for an incoming Relationship to create a Request out of the RelationshipTemplate that was used to create the Relationship and to directly complete the Request using the Response sent in the Relationship's creation content
 
```

Comment:
in the `creationContent` of the Relationship. 

Because @britsta told me to quote the properties...

Reply:
I think in this case it's fine because it's more description about what happens and not talking about a property

Reply:
Delaying this since it will be part of a bigger discussion

Reply:
as most of the places use `creationContent` we should maybe align it until we discuss it later.